tests/shared/src/test/scala/cats/tests/SyntaxSuite.scala

```diff
@@ -544,6 +545,70 @@ object SyntaxSuite {
     val nested: Nested[F, G, A] = fga.nested
   }
 
+  def testEither[F[_]: Applicative, A: Order, B: Order](): Unit = {
+    val either = mock[Either[A, B]]
+    val a = mock[A]
+    val b = mock[B]
+
+    val v1 = either.toValidated
+    val v2 = either.toValidatedNel
+    val v3 = either.toValidatedA[F]
+
+    val v4 = either.toEitherA
+    val v5 = either.toEitherT
+    val v6 = either.toEitherNel
+    val v7 = either.toEitherNec
+    val v8 = either.toEitherNes
+
+    val v9 = Either.left[A, B](a)
+    val v10 = Either.right[A, B](b)
+
+    val v11 = Either.leftNec(a)
+    val v12 = Either.rightNec(b)
+    val v13 = Either.leftNes(a)
+    val v14 = Either.rightNes(b)
+    val v15 = Either.leftNel(a)
+    val v16 = Either.rightNel(b)
+    val v17 = Either.leftA[F, A, B](a)
+  }
+
+  def testOption[F[_]: Applicative, G[_], A: Order, B: Order: Monoid](): Unit = {
```

Comment:
Sorry for my meticulousness, but where are these `A: Order` and `B: Order` constraints used? I can see their usage in the previous test, but not in this one (nor in the next one).